devZoGok/Battleship
# Ft 82 unit states

Implemented unit states. To test:
1) create a war mech and use `U` to enable the `HOLD_FIRE` state
2) move the war mech right next to a hostile building
3) observe the war mech remain idle
4) use `Y` to enable the `STAND_GROUND` state
5) observe the war mech attack the hostile building automatically
6) use `T` to enable the `CHASE` state
7) allow a hostile engineer move past the war mech
8) observe the war mech attack and pursue the engineer once it's entered the war mech's line of sight
